### ISSUE TITLE
[exam][speaking test]2023-03

### ISSUE THREAD

**Issue by coffee-qm** [coffee-qm/toeic#5]
[toc]

##### Text 1

Welcome to USC Career Center, where we help people find jobs. Today, I'd like to introduce our new employee database. You can use this database to search for jobs, receive updates, or even submit your resume with a single click. Moreover, the resources and database can help you promote yourself to employers who may be interested in your work skills.

???
Thank you for attending this press conference. It's my pleasure to present our company's new director, Linda Smith. Ms. Smith focus on expanding our client base, supporting your employees, and improving our products. Since she has over thirty years' experience in our field, I'm confident she will provide valuable leadership. Now, please welcome Ms. Linda Smith.

##### Text 2

Thanks for joining us at Community Radio. Our next guest is Dr. Naomi Evans, who has a new book about maintaining a balanced lifestyle. During her career, Dr. Evans has focused her lifestyle on general well-being. She has authored numerous books about expecise, eating habits, and relieving stress. Dr. Evans, thanks for coming today!

???
Are you thinking of taking a trip? Let the team at Travel Smart help you find the best deals on airline tickets, hotel reservations, and tours. For more than thirty years, our agency has been providing personalized service and convenience to travelers. Whether your destination is international or domestic, we can make your trip a success.

##### Picture 1

<img width="800" alt="TOEIC_SpeakingTest_2023-03_part2_1" src="https://github.com/coffee-qm/toeic/assets/32775335/c1edbf98-3aa0-4c5a-8628-022bb3520d6e">

This is the picture taken at an office.
In the middle, there is a woman with long hair. She is wearing a sleeveless white shirt and dark pants. She is standing at the table while looking down some documents placed on the table. She is holding a piece of paper in one hand. It looks like she is focusing on reading handouts.
On the table, I can see a keyboard, a lamp and other stationeries. Behind her, there is a shelf with some items arranged on it.

##### Picture 2

<img width="854" alt="TOEIC_SpeakingTest_2023-03_part2_2" src="https://github.com/coffee-qm/toeic/assets/32775335/8f182984-b36e-4f39-8922-2bace2ca754a">

This is the picture taken at a meeting room.
In the middle, there is a woman with short curly hair. She is wearing a white blazer and dark pants. She is standing in the middle while sticking a note on the board. She is holding another note in the other hand. It looks like she is presenting some ideas.
In front of her, there are some other people sitting around the table. They are all watching the board and listening the presenter.
Overall, people are having a meeting.

##### Topic: Cooking habits

Imagine that a cooking magazine is conducting a survery on cooking habits. You have agreed to join a telephone interview regarding cooking habits.

Q: When was the last time you cooked? What did you make?
A: The last time I cooked was the last weekend. I made spaghetti for my family gathering. We all love this delicious dish.

Q: How do you usually get recipes when you are cooking a new dish? Why?
A: Once I want to try a dish I have never made, I often get the recipes through the Internet. It is easy to access instruction videos from my smartphone. I can see the cooking process while I make.

Q: Do you like reading food magazines? Why or why not?
A: Yes, I definitely love food magazines. They give me the chance to try out some new recipes. Through the articles appearing in the magazines, I can learn about the nutrition percentage of that dish. In addition, food pictures throughout the magazines are one of the primary reasons I am into the journal. The photographs are so beautiful and motivate me to make that dish. However, some of the ingredients listed in the instruction are not easy to find. Overall, food magazines are useful and motivational to me.

##### Topic: Festival Tours

<img width="912" alt="TOEIC_SpeakingTest_2023-03_part4" src="https://github.com/coffee-qm/toeic/assets/32775335/0b51e984-f020-4c38-aa3f-68f8ccb2a359">

Hi, my name is Lucy. I want to participate in High Elevation Rock Festival Tour. Can I ask you some questions before I buy the ticket?

Q: Do you offer any discount to attendees? And how can I get it?
A: Hello, I am so glad that you are interested in this event. Well, you can be eligible for 5$ off if you buy tickets online.

Q: All the performances are scheduled on Saturdays, right?
A: Well, I'm afraid not. There is one performance taking place on Friday, and the others are held on Saturdays.

Q: I am eager to enjoy the festival scheduled in New York City. Can you give me detailed information about them?
A: Well, there are two performances held in New York City. 
The first one will perform on Saturday, March 24th at New York City Dent Hall. But unfortunately, there are no tickets left. 
The second one will take place on Saturday, April 14th at Zeppelin Hall. And you can pay 27$ for each ticket.

##### Topic: Travel, alone or with other people

Q: Do you prefer to travel alone or with other people?
A: 
Traveling is one of the most effective ways to relieve stress. I prefer to travel with other people like my friends, my colleagues, or my family.
Firstly, enjoying travel with other people can create valuable memories. We can take pictures together to keep happy experience. Also, traveling together can develop the relationship among members through stories we share during the trip.
Moreover, traveling with many people can save money thanks to sharing expenses of transportation, accommodations, and meals. Members can also help and provide safety to each other when facing common difficulties in a strange place.
In conclusion, there are many benefits of traveling with other people including sharing costs, improving safety, and keeping memories.
